doc/INSTALL-Ubuntu-22.04.md

```diff
@@ -0,0 +1,170 @@
+# Installing Aleph-VM on a server / Ubuntu 22.04 Jammy Jellyfish
+
+## 0. Introduction
+
+For production using official Debian packages.
+
+## 1. Requirements
+
+- A [supported Linux server](../vm_supervisor/README.md#1-supported-platforms)
+- A public domain name from a registrar and top level domain you trust. 
+
+In order to run an official Aleph.im Compute Resource Node (CRN), you will also need the following resources:
+
+- CPU (2 options):
+  - Min. 8 cores / 16 threads, 3.0 ghz+ CPU (gaming CPU for fast boot-up of microVMs)
+  - Min. 12 core / 24 threads, 2.4ghz+ CPU (datacenter CPU for multiple concurrent loads)
+- RAM: 64GB
+- STORAGE: 1TB (NVMe SSD preferred, datacenter fast HDD possible under conditions, you’ll want a big and fast cache)
+- BANDWIDTH: Minimum of 500 MB/s
+
+You will need a public domain name with access to add TXT and wildcard records.
+
+> 💡 This documentation will use the invalid `vm.example.org` domain name. Replace it when needed.
+
+## 2. Installation
+
+Run the following commands:
+
+First install the [VM-Connector](../vm_connector/README.md) using Docker:
+```shell
+sudo apt update
+sudo apt upgrade
+sudo apt install -y docker.io
+docker run -d -p 127.0.0.1:4021:4021/tcp --restart=always --name vm-connector alephim/vm-connector:alpha
```

Comment:
Unrelated, but we should really version the connector image.

Reply:
And move it into a VM to remove the dependency on Docker